tests/test_browser.py

```diff
@@ -3295,164 +3295,151 @@ def prep_no_SAB(self):
 
   # Test that the emscripten_ atomics api functions work.
   def test_pthread_atomics(self):
-    self.btest(path_from_root('tests', 'pthread', 'test_pthread_atomics.cpp'), expected='0', args=['-s', 'TOTAL_MEMORY=64MB', '-O3', '-s', 'USE_PTHREADS=2', '--separate-asm', '-s', 'PTHREAD_POOL_SIZE=8'], timeout=120) # extra time on first test, to be sure to build all libraries
+    self.btest(path_from_root('tests', 'pthread', 'test_pthread_atomics.cpp'), expected='0', args=['-s', 'TOTAL_MEMORY=64MB', '-O3', '-s', 'USE_PTHREADS=1', '-s', 'PTHREAD_POOL_SIZE=8'])
```

Comment:
Can you retain the timeout here? The `extra time on first test` comment here refers to the aspect that this is the first multithreaded test in the suite, which means that when it is hit, it will potentially trigger first build of `libc-mt` and other multithreaded runtime libraries, which can otherwise time out if not given enough time to run to completion.

Reply:
Good point. Another option could be to use embuilder to build libc-mt before running tests.

Reply:
I looked into into because it didn't make sense to me.  The timeout is not for the compile time.  The compile time not limited.  The timeout is only applys to the running of the browser after the compile has finished.   So the comment doesn't make sense to make (at least not in the current version btest()).

Reply:
Ah, that makes sense, we start the timeout when we run the browser portion, after compilation. So this change looks good (and worst case, we'd see a timeout on the bots).